doc/en/user/source/extensions/netcdf/netcdf.rst

```diff
@@ -53,16 +53,35 @@ Opening a sample NetCDF using that tool will show an output like this in the Vie
 * Each dimension has an associated independent coordinate variable (marked by the green rectangle).
 * Finally, the dataset has 3 geophysical variables, marked by a red rectangle, each having 4 dimensions.
 
-The NetCDF plugin supports datasets where each variable's axis is identified by an independent coordinate variable, as shown in the previous example.
-Two dimensional non-independent latitude-longitude coordinate variables aren't currently supported. 
-A coordinate variable like this will be a coordinate variable having 2 dimensions to index it (such as y,x/rows,cols/i,j...).
+The NetCDF plugin fully supports datasets where each variable's axis is identified by an independent coordinate variable, as shown in the previous example. There is limited support for two dimensional coordinate variables (see :ref:`netcdf_multidim`). A coordinate variable like this will be a coordinate variable having 2 dimensions to index it. This can be part of the definition of a plain dataset (such as y,x/rows,cols/i,j/...) or the result of an aggregation (such as time,runtime - in the case of a runtime aggregation).
```

Comment:
There seems to be redundancy in the last two sentences. Perhaps remove this sentence: "A coordinate variable like this will be a coordinate variable having 2 dimensions to index it." and the parenthetical remark "(such as y,x/rows,cols/i,j/...)"?